cmd/bb_runner/BUILD.bazel

```diff
@@ -65,6 +65,25 @@ container_image(
     visibility = ["//visibility:public"],
 )
 
+container_image(
```

Comment:
Do you actually need anything from the Alpine container image? Wouldn't it be sufficient to use a plain `go_image()` for this bare use case, just like we already do for bb_scheduler and bb_worker?

If you can't use `go_image()` literally, you can at least consider using `base = "@go_image_static//image"`, which we also use for bb_browser.

Reply:
Also be sure to add a `container_push_official()`, so that we provide ready to use container images. How should we call this? `bb-runner-bare`?